### PR TITLE
Update bundle_fr.properties

### PR DESCRIPTION
**Pull request by clfer**
Hi,

Love your game.
Made an update for the french translation.

Have a nice day

### CODE REVIEW

File: core/assets/bundles/bundle_fr.properties [hunk 1]
```diff
@@ -60,6 +60,18 @@ uploadingcontent = Publication du Contenu
 uploadingpreviewfile = Publication du Fichier d'Aperçu
 committingchanges = Validation des Modifications
 done = Fait
+
+mods.alphainfo = Gardez à l'esprit que les mods sont en alpha et{scarlet] peuvent être très buggés[].\nMerci de signaler les problèmes que vous rencontrez via le Github ou le Discord Mindustry.
```
Comment (amauryvallier) [inline core/assets/bundles/bundle_fr.properties]: {scarlet] is a typo, I guess ! :)

**Comment by runiArnsbjornson** [comment]
ok thanks for the update and report :) any feedback on missing translation or errors are welcomed

File: core/assets/bundles/bundle_fr.properties [hunk 2]
```diff
@@ -368,75 +402,80 @@ launch.confirm = Cela va transférer toutes les ressources de votre noyau.\nVous
...
 settings.controls = Contrôles
 settings.game = Jeu
 settings.sound = Son
 settings.graphics = Graphismes
 settings.cleardata = Effacer les données du jeu...
 settings.clear.confirm = Êtes-vous sûr d'effacer ces données ?\nAucun retour en arrière n'est possible!
-settings.clearall.confirm = [scarlet]ATTENTION![]\nCet action effacera toutes les données, y conpris les sauvegarges, les cartes, la progression et la configuration des touches.\nUne fois que vous aurez pressé 'ok' le jeu effacera TOUTES les données et se fermera.
-settings.clearunlocks = Effacer la progression
-settings.clearall = Tout effacer
+settings.clearall.confirm = [scarlet]ATTENTION![]\nCet action effacera toutes les données, y compris les sauvegardes, les cartes, la progression et la configuration des touches.\nUne fois que vous aurez pressé 'ok' le jeu effacera TOUTES les données et se fermera.
```
Comment (runiArnsbjornson) [inline core/assets/bundles/bundle_fr.properties]: Cette action

Comment (clfer) [reply core/assets/bundles/bundle_fr.properties]: Done
Thanks

**Pull request merged**